packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/legacy/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+    "datacenter_backup_legacy_message_1": "Depuis le 25/02/2020, la gamme Veeam Backup a évolué.",
```

Comment:
Expected only 2 space for indentation.
see: https://github.com/ovh/manager/blob/master/.editorconfig#L11-L12